mlflow/ml-package-versions.yml

```
@@ -213,24 +213,21 @@ gluon:
     run: |
       pytest tests/gluon_autolog/test_gluon_autolog.py --large
 
-fastai-1.x:
+fastai-2.x:
```

```suggestion
fastai:
```

I think we can remove `-2.x`

---

Looks like fastai-2.x is required.

---

I think we also need to fix this line:

https://github.com/mlflow/mlflow/pull/4715/files#diff-f7f812cf7b56d2001536c3463d4f4464ec4fa521f176b010dd1e28ed14b93bf4R13